projects/discussion-ui/src/lib/components/my-discussion/my-discussion.component.ts

```diff
@@ -195,7 +220,7 @@ export class MyDiscussionComponent implements OnInit {
     const userSlug = this.discussService.userDetails.userslug;
     combineLatest([
       this.discussService.fetchUserProfile(userId),
-      this.discussService.fetchRecentPost(userSlug, scrollIndex)
+      this.discussService.fetchRecentPost()
```

Comment:
@balakrishna10 
can u confirm if the recent post API supports an optional parameter as pageId or not ? it is needed for infinite scroll

Reply:
this is supported in CSL and scrollIndex is optional parameter.